### DISCUSSION TITLE
Gov Cloud API report downloads

### DISCUSSION THREAD

**Discussion by engineer330426** [CrowdStrike/falconpy#994] [Q&A] [answered]
I've tried using all of the following and nothing seems to have worked

https://github.com/CrowdStrike/falconpy/discussions/809
https://assets.falcon.laggar.gcw.crowdstrike.com/support/api/swagger-eagle.html#/report-executions/report-executions-download.get

all with the report ID's found in the urls and even coppied the IDs from the download links.  Nothing seems to work, does gov cloud just not have this functionality.

Also opened a support ticket

**Comment by jshcodes** [comment]
Are you still encountering this issue?  I'm currently unable to recreate it, but that could be related to my test environment.

**Reply by engineer330426** [reply, marked as answer]
no i found a way to get the results from scheduled searches.
